core/src/main/scala/cats/data/Ior.scala

```diff
@@ -375,6 +375,8 @@ sealed abstract class Ior[+A, +B] extends Product with Serializable {
 
   final def toIorNec[AA >: A]: IorNec[AA, B] = leftMap(NonEmptyChain.one)
 
+  final def toIorA[F[_]: Applicative, AA >: A]: Ior[F[AA], B] = leftMap(Applicative[F].pure)
```

Comment:
Sorry, I overlooked it previously. If this method is defined in such a way, that basically means that in most cases a user will have to repeat the `A` type every time when calling the method:
```scala
val myior: Ior[Foo, Bar] = ???
myior.toIorA[NonEmptyWhatever, Foo]
```
I doubt that it makes sense in most of cases. I think that in most cases users would prefer 
```scala
val myior: Ior[Foo, Bar] = ???
myior.toIorA[NonEmptyWhatever]
```
instead. Wdyt?

I think we discussed it for some other types already, so I believe it would be better to be consistent on such `toWhateverA` methods across the library.

Reply:
agree 👍

Reply:
actaully, the issue here is that `final def toIorA[F[_]: Applicative]: Ior[F[A], B] = leftMap(Applicative[F].pure)` can't be compiled because covariat type `A` is provided to invariant `F[_]`. Do you think it worth it to write somethinkg like `LiftToPartiallyApplied` as well?

Reply:
I did what you suggested, however, it results in `myior.toIorA[NonEmptyWhatever]()` with `()` to call the apply method

Reply:
Yes, I see that hurdle now, thank you for diving into it.
Well, a work-around that I've come up with is using `F[+_]` instead of just `F[_]`:
```scala
final def toIorA[F[+_]](implicit F: Applicative[F]): Ior[F[A], B] = leftMap(F.pure[A])
```
I tried it briefly and seems working for me.

It won't allow to parametrize it with, let's say, `NonEmptySet`, but... that type does not have `Applicative` defined anyway.

Reply:
However, I've just realized that functions like this one do always have the following shape:
```
LeftRightType[A, B] => Applicative[F] => LeftRightType[F[A], B]
```
and they are nothing but `leftMap(F.pure[A])`, aren't they?

Therefore, it is really possible to avoid defining duplicates of `to<LeftRightType>A` in each `LeftRightType` if we simply define just one function in `Bifunctor` instead:
```scala
trait Bifunctor[F[_, _]] {
  // the name is a subject for bike-shedding of course :)
  def leftMapA[A, B, C[_]](fab: F[A, B])(implicit C: Applicative[C]): F[C[A], B] =
    leftMap[A, B, C[A]](fab)(C.pure[A]) 
}
```
and its syntax of course.

All these types `Ior`, `Validated` and `Either` do have `Bifunctor` defined, btw.

So wdyt?

Reply:
@johnynek , could you share your thoughts on this please? What if we add just a single method to `Bifunctor` instead of multiple very similar methods in `Ior`, `Vaidated`, etc?

Reply:
(sorry for pinging, just to draw your attention to this thread)

Reply:
I like the idea of implementing a single method on Bifunctor!

Reply:
I am not sure about a name to choose for the new syntax. `leftMapA` looks a bit controversial to me.
I would think something about `left` and `lift` words combined. `leftLiftTo` maybe? Do you have any thoughts/preferences on that?

Reply:
I don't really have a preference, `leftLiftTo` sound good to me 👍

Reply:
Done🤚 Removed the syntax I added before, added `Bifunctor` method, and updated tests

Reply:
Thank you for your efforts! To me personally it looks way cleaner now.